Azure/communication-ui-library-ios
# [Feature][DemoApp] Add XCUITest inside Demo App

## Purpose

* Add AzureCommunicationUIDemoAppUITests  as test target inside AzureCommunicationUIDemoApp
* Add `info.plist` to configure environment variables inside UI tests ( only add `expiredAcsToken` as an example for now, you need to add this variable inside local `EnvConfig.xcconfig` file)
* Add `XCUITestBase` as the base class
* Pod `import SwiftLint` for AzureCommunicationUIDemoAppUITests
* Some basic UI tests setup 

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  See if you can run the local UI test after adding `expiredAcsToken` to your local `EnvConfig.xcconfig` file 
See the steps on the [Running UI Test](https://github.com/Azure/communication-ui-library-ios/blob/feature/uitest/docs/contributing-guide.md#running-ui-tests)
